### PR TITLE
Add support for class member variable

### PR DESCRIPTION
**Pull request by vicb**
I am looking for:
- feedback on the implemenation,
- how to debug (currently I'm doing make + node debug which is not the most user friendly workflow),
- what kind of test should be added.

/cc @arv

### CODE REVIEW

File: src/syntax/Parser.js [hunk 1]
```diff
@@ -2170,6 +2171,13 @@ export class Parser {
         annotations, body);
   }
 
+  parsePropertyVariableDeclaration_(start, isStatic, name, annotations) {
+    var typeAnnotation = this.parseTypeAnnotationOpt_();
+    this.eat_(SEMI_COLON);
```
Comment (caitp) [inline src/syntax/Parser.js]: doesn't `TypeAnnotationopt` indicate that the type annotation is optional?

Comment (vicb) [inline src/syntax/Parser.js]: I think the comment is on `this.eat_(SEMI_COLON);` / ";" 
No question typeAnnotation is optional.

Comment (vicb) [inline src/syntax/Parser.js]: I think the comment is on `this.eat_(SEMI_COLON);` / ";"

Comment (caitp) [inline src/syntax/Parser.js]: oh you're right, I misread the symbol.

**Pull request closed without merging**